{"uid":"530c132132d71cb2","name":"test_setup","fullName":"tests.integration.test_smoke#test_setup","historyId":"d623373ae070a62cab8e4b2f9dd522ae","time":{"start":1750469056951,"stop":1750469061649,"duration":4698},"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1750469056727,"stop":1750469056727,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1750469056727,"stop":1750469056727,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1750469056728,"stop":1750469056950,"duration":222},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1750469056951,"stop":1750469056951,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1670, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 156, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 31, in test_setup\n    await ops_test.model.deploy(\"./releases/latest/postgresql-k8s-bundle.yaml\", trust=True)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 1914, in deploy\n    await handler.execute_plan()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 453, in execute_plan\n    await self._resolve_charms()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/bundle.py\", line 436, in _resolve_charms\n    charm_url, charm_origin = await self.model._resolve_charm(\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 2078, in _resolve_charm\n    resp = await charms_facade.ResolveCharms(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 482, in wrapper\n    reply = await f(*args, **kwargs)\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/_client6.py\", line 276, in ResolveCharms\n    reply = await self.rpc(msg)\n            ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/facade.py\", line 680, in rpc\n    result = await self.connection.rpc(msg, encoder=TypeEncoder)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/client/connection.py\", line 597, in rpc\n    raise errors.JujuAPIError(result)\njuju.errors.JujuAPIError: transforming slice at index 0: cannot parse name and/or revision in URL \"\": name \"\" not valid","steps":[],"attachments":[{"uid":"f8e9388fc8631c47","name":"log","source":"f8e9388fc8631c47.txt","type":"text/plain","size":1095}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1750469061662,"stop":1750469061662,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1750469061663,"stop":1750469061973,"duration":310},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1750469061973,"stop":1750469061973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1750469061973,"stop":1750469061973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1750469061973,"stop":1750469061973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"fv-az1979-727"},{"name":"thread","value":"16591-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"2a0e5e714b31a7d5","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":29,"skipped":0,"passed":319,"unknown":9,"total":357},"items":[{"uid":"129b0999e298a272","reportUrl":"../2134//#testresult/129b0999e298a272","status":"passed","time":{"start":1750382670407,"stop":1750383254437,"duration":584030}},{"uid":"b72bfb0c2059ead8","reportUrl":"../2133//#testresult/b72bfb0c2059ead8","status":"passed","time":{"start":1750296241156,"stop":1750296813314,"duration":572158}},{"uid":"6f2b8a8ed946ee46","reportUrl":"../2132//#testresult/6f2b8a8ed946ee46","status":"passed","time":{"start":1750209851354,"stop":1750210436584,"duration":585230}},{"uid":"e0b56891da3d5bf9","reportUrl":"../2129//#testresult/e0b56891da3d5bf9","status":"passed","time":{"start":1750123520920,"stop":1750124143127,"duration":622207}},{"uid":"61195258665448e6","reportUrl":"../2128//#testresult/61195258665448e6","status":"passed","time":{"start":1750037038110,"stop":1750037660753,"duration":622643}},{"uid":"9d124cd8c479c360","reportUrl":"../2127//#testresult/9d124cd8c479c360","status":"passed","time":{"start":1749950683210,"stop":1749951305678,"duration":622468}},{"uid":"9f3ddd83fdf343b5","reportUrl":"../2126//#testresult/9f3ddd83fdf343b5","status":"passed","time":{"start":1749864344595,"stop":1749864961447,"duration":616852}},{"uid":"efbc3172f8f3441","reportUrl":"../2125//#testresult/efbc3172f8f3441","status":"passed","time":{"start":1749777823526,"stop":1749778450316,"duration":626790}},{"uid":"a529ef84e36ca775","reportUrl":"../2124//#testresult/a529ef84e36ca775","status":"passed","time":{"start":1749691447163,"stop":1749692022413,"duration":575250}},{"uid":"4a1c1ca24a3332f1","reportUrl":"../2123//#testresult/4a1c1ca24a3332f1","status":"broken","statusDetails":"juju.errors.JujuAppError: App in error: postgresql-k8s","time":{"start":1749605141765,"stop":1749605204891,"duration":63126}},{"uid":"3c3fab00a0a6628f","reportUrl":"../2120//#testresult/3c3fab00a0a6628f","status":"passed","time":{"start":1749518643394,"stop":1749519178498,"duration":535104}},{"uid":"a77bc2929768b2af","reportUrl":"../2119//#testresult/a77bc2929768b2af","status":"passed","time":{"start":1749432284170,"stop":1749432848798,"duration":564628}},{"uid":"242d24a8fd26c705","reportUrl":"../2118//#testresult/242d24a8fd26c705","status":"passed","time":{"start":1749345869145,"stop":1749346508601,"duration":639456}},{"uid":"160283475727080","reportUrl":"../2117//#testresult/160283475727080","status":"passed","time":{"start":1749259421302,"stop":1749260070431,"duration":649129}},{"uid":"116e59ec09bb23ea","reportUrl":"../2115//#testresult/116e59ec09bb23ea","status":"passed","time":{"start":1749173133551,"stop":1749173759496,"duration":625945}},{"uid":"a0e51b1cffe25055","reportUrl":"../2114//#testresult/a0e51b1cffe25055","status":"passed","time":{"start":1749086609091,"stop":1749087192694,"duration":583603}},{"uid":"fbd29f426f828a2d","reportUrl":"../2113//#testresult/fbd29f426f828a2d","status":"passed","time":{"start":1749000234697,"stop":1749000850406,"duration":615709}},{"uid":"37e7f3807ca6a8b7","reportUrl":"../2110//#testresult/37e7f3807ca6a8b7","status":"passed","time":{"start":1748913919370,"stop":1748914582442,"duration":663072}},{"uid":"60174688880f1487","reportUrl":"../2109//#testresult/60174688880f1487","status":"passed","time":{"start":1748827474762,"stop":1748828173688,"duration":698926}},{"uid":"c399d86f3c5a3223","reportUrl":"../2108//#testresult/c399d86f3c5a3223","status":"passed","time":{"start":1748741123297,"stop":1748741784519,"duration":661222}}]},"tags":["asyncio","abort_on_fail"]},"source":"530c132132d71cb2.json","parameterValues":[]}